pom.xml

```diff
@@ -73,8 +68,8 @@
         <module>test/service-test-utils</module>
         <module>test/test-utils</module>
         <module>test/codegen-generated-classes-test</module>
-        <module>annotations</module>
         <module>utils</module>
+        <module>core</module>
```

Comment:
nit, can we move `core` in the top and move `utils` before test modules? It's nice to separate non-test modules and test modules.